### ISSUE TITLE
Add text description of OID in artifact names.

### ISSUE THREAD

**Issue by johngray-dev** [IETF-Hackathon/pqc-certificates#96]
When looking at the artifacts in an artifact.zip file, it is difficult to know what the underlying algorithm is used unless you just happen to know the OID associated with that algorithm.   It would be nice to have something like:
- <OID>-<StringName>.der so it was easier to quickly find samples of the desired algorithm type.
- This has been mentioned a few times by various people using the project.

**Comment by danvangeest** [comment]
In the OID mapping table, the value which would be used by `StringName` would have dashes in it, e.g. "ML-DSA-44-ipd", so I think an underscore in the name is better, e.g [OID]_[StringName][OtherArtifactDifferentiators].der.

This is what I'll be doing in the forthcoming CMS artifacts.

**Comment by danvangeest** [comment]
Of course, right after writing this I realized that kyber512_shake has an underscore in it.

**Comment by johngray-dev** [comment]
This should be done in the V4 format version.
